### ISSUE TITLE
Add more Wizards

### ISSUE THREAD

**Issue by deingithub** [The-Valley-Discord/blimp#2]
Should guide users through setting up and, if necessary, modifying:

- [x] Role Kiosks
- [ ] Welcome&Goodbye
- [x] Boards

Should offer related features (eg open triggers for tickets).

Should issue commands automatically after user reaction, using `ctx.invoke_command`.

**Comment by deingithub** [comment]
Further ideas:

- [ ] Welcome&Goodbye wizards (low priority because the commands are fairly simple)
- [ ] "Autokiosk" that has an attached `post!` that gets edited automagically when you edit the attached kiosk with the wizard
- [ ] Open triggers for ticket categories, possibly similar to "autokiosk"

**Comment by deingithub** [comment]
The "autokiosk" mentioned above, under the name role menu, will be the main feature of version 2.5.

**Comment by deingithub** [comment]
> The "autokiosk" mentioned above, under the name role menu, will be the main feature of version 2.5.

This may be a lie. We'll instead wait for components to arrive in discord.py before starting a rewrite and redesign of the current wizard and kiosk setup.
